Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealityDeviceManager.cs

```diff
@@ -721,7 +721,7 @@ private BaseWindowsMixedRealitySource GetOrAddController(InteractionSource inter
             }
 
             string nameModifier = controllingHand == Handedness.None ? interactionSource.kind.ToString() : controllingHand.ToString();
-            bool isHPController = !interactionSource.supportsTouchpad;
+            bool isHPController = !interactionSource.supportsTouchpad && interactionSource.kind == InteractionSourceKind.Controller;
             string inputSourceName = isHPController ? $"HP Motion Controller {nameModifier}" : $"Mixed Reality Controller {nameModifier}";
```

Comment:
Probably out of scope of this PR, but do we want to add a third name for hands? `Mixed Reality Hand {nameModifier}` or something?

Reply:
That's a good idea, will do